i18n/extra/es.json

```diff
@@ -93,8 +93,8 @@
             "SMW_YM"
         ],
         [
-            "SMW_MDY",
             "SMW_DMY",
+            "SMW_MDY",
```

Comment:
>  I speak Portuguese, not Spanish

Dammit and for this faux pas I'd probably get arrested from the new Dumpf administration for cultural insensitivity.